cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -59,7 +58,7 @@ import org.ossreviewtoolkit.utils.safeDeleteRecursively
 import org.ossreviewtoolkit.utils.showStackTrace
 
 class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source code from a remote location.") {
-    private val input by mutuallyExclusiveOptions<GroupTypes>(
+    private val input by mutuallyExclusiveOptions(
```

Comment:
Note: I recall these were already flagged as unneeded by the IDE before we switched Kotlin 1.4, but in fact they were needed else compilation failed. But maybe with Kotlin 1.4 they're indeed not needed anymore.

Reply:
This can be compiled now it seems.

Reply:
Great, how about a hint in the commit message that this only works as of Kotlin 1.4?

Reply:
Done.